omod/src/main/java/org/openmrs/module/coreapps/page/controller/MarkPatientDeadPageController.java

```diff
@@ -37,7 +37,7 @@ public void get(@SpringBean PageModel pageModel, @RequestParam(value = "breadcru
         pageModel.put("patientId", patient.getId());
         pageModel.put("breadcrumbOverride", breadcrumbOverride);
         if (conceptId != null && !conceptId.contains("[a-zA-Z]+")) {
```

Comment:
Since you now allow uuid and conceptId, do we still need this? conceptId.contains("[a-zA-Z]+

Reply:
removed this check :)